sympy/codegen/array_utils.py

```diff
@@ -435,22 +435,6 @@ def _get_contraction_links(self):
         args, dlinks = _get_contraction_links([self], self.subranks, *self.contraction_indices)
         return dlinks
 
-    @staticmethod
-    def from_MatMul(expr):
```

Comment:
Should this follow the deprecation cycle?

Reply:
No, it wasn't working properly. I doubt people were using it.